### PR TITLE
Update docs for meteor/apollo v3

### PR DESCRIPTION
**Pull request by lorensr**
v3: https://github.com/apollographql/meteor-integration/pull/124 

Will release when this is merged. Maybe there's a better way to connect to `WebApp`, maybe @abernix knows? Currently recommending:

```js
const server = new ApolloServer({...})

server.applyMiddleware({
  app: WebApp.connectHandlers,
  path: '/graphql'
})

// We are doing this work-around because Playground sets headers and WebApp also sets headers
// Resulting into a conflict and a server side exception of "Headers already sent"
WebApp.connectHandlers.use('/graphql', (req, res) => {
  if (req.method === 'GET') {
    res.end()
  }
})
```

### CODE REVIEW

**Comment by hwillson** [comment]
Thanks for this @lorensr! Actually, we've been talking about the Meteor integration docs a fair bit recently. Given that you're planning on releasing a 3.0.0 version of https://github.com/apollographql/meteor-integration, does that mean you (or someone else) will be supporting/maintaining it? We were under the impression that it is currently unmaintained (due to the repo's README), and we were planning on completely removing most of Apollo's Meteor integration section. We were going to instead replace that full section with a single page, that links to modern and maintained community approaches for integrating Meteor and Apollo. As things stand currently, our preference is to point people towards using https://github.com/Swydo/ddp-apollo, since it's actively maintained. If someone is taking over the `apollo` package though, great!

So long story short - if the `apollo` package is going to be maintained by someone, we'll definitely get these changes merged in. If it's not though, we're likely moving ahead with reducing this section greatly. Let me know - thanks!

**Comment by lorensr** [comment]
Happy to keep maintaining it, and to direct people to @swydo—lmk if there
are other differences I should note in the beginning of the docs

**Comment by hwillson** [comment]
Okay, thanks @lorensr. I'll add a few adjustments here (and will likely remove the subscriptions section completely since `SubscriptionManager` has been deprecated), then get this in place. 

@theodorDiaconu I know you're doing a lot of interesting things with regards to Meteor + Apollo integration. Once we have this section cleaned up a bit, if you're interested in adding a reference to your https://github.com/cult-of-coders/apollo project, let me know. Thanks!

**Comment by hwillson** [comment]
@theodorDiaconu I just meant that in the [Meteor + Apollo section](https://www.apollographql.com/docs/react/recipes/meteor.html), we're mentioning the `apollo` package and [`swydo:ddp-apollo`](https://github.com/Swydo/ddp-apollo). If you would like us to add a link to your https://github.com/cult-of-coders/apollo project as well, let us know.

**Comment by lorensr** [comment]
@hwillson v3 of `meteor/apollo` has been published

**Comment by tab00** [comment]
Good work with the new content so far, guys. It has been much needed.

You can remove the instructions to install `graphql-playground` as it's now automatically installed. You can simply load it in the browser with `/graphql` path.

Please update the **Subscriptions** section to show new syntax for **Subscriptions** using https://github.com/apollographql/subscriptions-transport-ws (with websocket link, without `SubscriptionManager`, and with `SubscriptionServer`), as I've been having trouble getting it working and have been looking around.

Thank you.

**Comment by tab00** [comment]
Can we see some working code for using the built-in subscription server in Apollo Server? I've been having some trouble. See my code at https://github.com/apollographql/apollo-server/issues/1537

**Comment by hwillson** [comment]
I'm going to merge this as is. @tab00, any chance you'd like to create a separate PR with the subscription section changes? Thanks all!
